src/ptf/dataplane.py

```diff
@@ -949,7 +956,7 @@ def flush(self):
         """
         Drop any queued packets.
         """
-        for port_id in self.packet_queues.keys():
+        for port_id in list(self.packet_queues.keys()):
```

Comment:
A whole bunch of these "casts" to lists, especially for simple iterations are an overkill.